test/celeritas/Constants.test.cc

```diff
@@ -86,7 +86,8 @@ TEST(ConstantsTest, derivative)
 {
     // Compared against definition of Dalton, table 8 of SI 2019
     EXPECT_SOFT_NEAR(1.66053906660e-27 * units::kilogram, atomic_mass, 1e-11);
-    EXPECT_SOFT_NEAR(1.602176634e-19, e_electron * units::volt, 1e-9);
+    EXPECT_SOFT_NEAR(
+        1.602176634e-19 * units::joule, e_electron * units::volt, 1e-9);
```

Comment:
Now we should be able to keep the original tolerance (but doesn't really matter).
```suggestion
        1.602176634e-19 * units::joule, e_electron * units::volt, 1e-11);
```

Reply:
Oh my god, I see what you mean now. I forgot that a unit energy in CGS is erg, not Joule. Jeez, when I read
```
/Users/seth/.local/src/celeritas-temp/test/celeritas/Constants.test.cc:89: Failure
Value of: e_electron * units::volt
  Actual: 1.6021766340000001e-12
Expected: 1.602176634e-19
Which is: 1.6021766339999999e-19
(Absolute error 1.6021764737823368e-12 exceeds tolerance 1e-14)
```
I missed the exponent actual and expected magnitudes were so different 😂: I was looking at the 9 decimal places of the expected value. I must have done this the first time through when I bumped the tolerance to 1e-11—deceptive for a number that small.

Reply:
lol it confused me too, don't worry!